src/celeritas/em/msc/UrbanMscHelper.hh

```diff
@@ -120,14 +122,16 @@ CELER_FUNCTION real_type UrbanMscHelper::calc_msc_mfp(Energy energy) const
 
 //---------------------------------------------------------------------------//
 /*!
- * Calculate the total energy loss over a given step length.
+ * Calculate the energy of a track that is stopped after the given step.
```

Comment:
I am lost here as this sounds like that the track is `stopped` at the end of step.

Reply:
Yeah I reworded this poorly: this is the energy of a hypothetical particle that, if it moves a distance `step`, has slowed down to zero energy at the end of the step.  Maybe we should just rename this to `calc_inverse_range`?